app/src/main/java/dev/rodosteam/questtime/utils/Languages.kt

```diff
@@ -0,0 +1,20 @@
+package dev.rodosteam.questtime.utils
+
+enum class Languages(val label: String, val code: String) {
+    ENGLISH("English","en"),
+    RUSSIAN("Русский","ru"),
+    DEUTSCH("Deutsch","de");
```

Comment:
Все-таки в коде мы используем английский, так что предлагаю

```suggestion
    GERMAN("Deutsch","de");
```